docs/site/Repositories.md

```diff
@@ -291,7 +291,108 @@ Please See [Testing Your Application](Testing-Your-Application.md) section in
 order to set up and write unit, acceptance, and integration tests for your
 application.
 
-## Persisting Data without Juggler [Using MySQL database]
+## Access KeyValue Stores
+
+We can now access key-value stores such as [Redis](https://redis.io/) using the
+[KeyValueRepository]((https://github.com/strongloop/loopback-next/blob/master/packages/repository/src/repositories/kv.repository.ts).
+
+### Define a KeyValue Datasource
+
+We first need to define a datasource to configure the key-value store. For
+better flexibility, we spilt the datasource definition into two files. The json
+file captures the configuration properties and it can be possibly overridden by
+dependency injection.
+
```

Comment:
sorry to chime in, but wouldn't be good to add the following NOTE?.  Especially for new LB4 adopters?

 **NOTE:** 
The prefix **redis** is related to the name of the data source, so you can use any name you want here. Usually you use `lb4 datasource name` to generate it, where `name`  is the prefix mentioned above.

Reply:
The effective `name` is from https://github.com/strongloop/loopback4-example-shopping/blob/master/src/datasources/redis.datasource.json#L2.

Reply:
👍 I didn't notice the context of the example-shopping.